uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Form/AbstractDrupalInteractionTaskForm.php

```diff
@@ -42,7 +42,7 @@ private function initContactDetails() {
     $emailParams = [
       'contact_id' => '$value.id',
       'return' => ['email'],
-      'is_primary' => 1,
```

Comment:
no need to change this but for next .. It is recommended to keep trailing commas